src/main/groovy/io/cify/framework/core/CifyExpectedConditions.groovy

```diff
@@ -0,0 +1,42 @@
+package io.cify.framework.core
+
+import org.openqa.selenium.Point
+import org.openqa.selenium.StaleElementReferenceException
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedCondition
+
+class CifyExpectedConditions {
```

Comment:
Does it make sense rather extend "org.openqa.selenium.support.ui.ExpectedCondition" ?

Reply:
org.openqa.selenium.support.ui.ExpectedCondition works a bit differently and often is not as reliable

Reply:
@tummima could you please elaborate a bit? What do you under "not reliable"?

Reply:
First org.openqa.selenium.support.ui.ExpectedCondition can be applied only to certain cases, you can try this in test code, when you import the library, it is not as flexible, thus custom conditions need to be made. Second "works a bit differently" means that they are made for more common cases and when using them we should fully understand what they do, thus when implementing it is easy to make the code "not reliable".

Reply:
> Does it make sense rather extend "org.openqa.selenium.support.ui.ExpectedCondition" ?

I does, but "org.openqa.selenium.support.ui.ExpectedCondition" has private constructor=(